lib/Wikia/src/Service/User/Attributes/UserAttributes.php

```diff
@@ -51,11 +51,11 @@ public function getAttributes( $userId ) {
 	public function getAttribute( $userId, $attributeName, $default = null ) {
 		$attributes = $this->loadAttributes( $userId );
 
-		if ( !is_null( $attributes[$attributeName] ) ) {
+		if ( isset( $attributes[$attributeName] ) ) {
```

Comment:
We need to make sure that the key both exists and is not null, which conveniently enough is exactly what `isset` does: http://php.net/manual/en/function.isset.php